### DISCUSSION TITLE
Freezing when text-to-speech is used

### DISCUSSION THREAD

**Discussion by Awesomoe4000** [lnreader/lnreader#421] [Q&A]
Anyone else having problems with the auto reader option? Last updates working version for me has been v.1.1.8, any update that just freezes the screen and i can't go to the next chapter or pull up the settings screen, it lets me go to the phones' home screen and do other things. Im on a one+7 pro and wonder if it's a compatibility problem.

**Comment by rajarsheechatterjee** [comment]
By auto reader do you mean Auto Scrolling?

It works fine for me in the latest stable release on my OnePlus Nord.

**Reply by Awesomoe4000** [reply]
I mean the option that reads it to you. The speaker icon at the top right  of the app when you select a chapter, next to the bookmark icon. It works awesome for keeping up with your novels while driving or working.

**Reply by rajarsheechatterjee** [reply]
Text to speech also works fine for me in the latest stable release. 

Does it happen for every chapter or only for a particular chapter? Also does text to speech works for you in other apps?

**Reply by Awesomoe4000** [reply]
It does it on every chapter of every novel. It will read the chapter fine but i will have to restart the app afterwards since it doesn't let me go to the next chapter.  Even v1.1.8 seems to do the same but only on very few novels. Text to speech only gives me problems on this app with the versions after v1.1.8, versions before worked fine.
